/asr_sdm_hardware:
  ros__parameters:
    uart_can:
      uart_port: "/dev/ttyS3"
      uart_baudrate: 115200